Scripts/ParseEmailFiles/ParseEmailFiles.py

```diff
@@ -3222,8 +3222,9 @@ def data_to_md(email_data, email_file_name=None, parent_email_file=None, print_o
     if 'HTML' in email_data:
         md += u"* {0}:\t{1}\n".format('Body/HTML', email_data['HTML'] or "")
 
-    md += u"* {0}:\t{1}\n".format('Attachments', email_data.get('Attachments') or "")
-    md += u"\n\n" + tableToMarkdown("Headers", email_data['HeadersMap'])
+    md += u"* {0}:\t{1}\n".format('Attachments', email_data['Attachments'] or "")
+    md += u"\n\n" + tableToMarkdown("Headers", email_data['Headers']).decode("utf-8", "ignore")
+    md += u"\n\n" + tableToMarkdown("HeadersMap", email_data['HeadersMap']).decode("utf-8", "ignore")
```

Comment:
Is the data encoded? what returns from email_Data['HeadersMap']?

Reply:
Moreover - how is this related to the Msg format support?

Reply:
Youre right, removed it